src/shared/views/Mapper/index.js

```diff
@@ -217,89 +220,84 @@ class _Mapper extends React.Component<Props, State> {
 
     renderIntroModal(creditString: string) {
         /* eslint-disable global-require */
-        const { tutorial } = this.props;
+        const { t, tutorial } = this.props;
         const { ...otherProps } = this.props;
         const projectObj = otherProps.navigation.getParam('project', false);
-        let comp;
+        let twoTaps;
 
         if (projectObj.projectType === 4) {
-            comp = (
-                <Text style={{ color: 'rgb(237, 209, 28)' }}>INCOMPLETE</Text>
-            );
+            twoTaps = { twoTaps: t('incomplete') };
         } else {
-            comp = <Text style={{ color: 'rgb(237, 209, 28)' }}>MAYBE</Text>;
+            twoTaps = { twoTaps: t('maybe') };
         }
 
         let content;
         if (!tutorial) {
             content = (
                 <>
-                    <Text style={styles.header}>How To Contribute</Text>
+                    <Text style={styles.header}>{t('instructions1')}</Text>
                     <View style={styles.tutRow}>
                         <Image
                             source={require('../assets/tap_icon.png')}
                             style={styles.tutImage}
                         />
-                        <Text style={styles.tutText}>TAP TO SELECT</Text>
+                        <Text style={styles.tutText}>{t('instructions2')}</Text>
                     </View>
                     <Text style={styles.tutPar}>
-                        Look for features listed in your mission brief. Tap each
-                        tile where you find what you&apos;re looking for. Tap
-                        once for&nbsp;
-                        <Text style={{ color: 'rgb(36, 219, 26)' }}>YES</Text>,
-                        twice for&nbsp;
-                        {comp}, and three times for&nbsp;
-                        <Text style={{ color: 'rgb(230, 28, 28)' }}>
-                            BAD IMAGERY (such as clouds)
-                        </Text>
-                        .
+                        <Trans
+                            i18nKey="Tutorial:instructions3"
```

Comment:
again here it looks like part of the tutorial have been put into the app.

Reply:
Same here, this is the help screen that is always the same.